tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/context/BMainInstance.java

```diff
@@ -586,26 +606,21 @@ public String runMainAndReadStdOut(String command, String[] args, Map<String, St
             }
 
             String[] cmdArgs = Stream.concat(Arrays.stream(cmdArray), Arrays.stream(args)).toArray(String[]::new);
-            ProcessBuilder processBuilder = new ProcessBuilder(cmdArgs).directory(new File(commandDir));
+            ProcessBuilder processBuilder =
+                    new ProcessBuilder(cmdArgs).directory(new File(commandDir)).redirectErrorStream(true);
             addJavaAgents(envProperties);
             Map<String, String> env = processBuilder.environment();
             env.putAll(envProperties);
 
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            PrintStream out = new PrintStream(baos);
             Process process = processBuilder.start();
-
-            // Give a small timeout so that the output is given.
-            Thread.sleep(5000);
```

Comment:
Is this not required now?

Reply:
It seems like not needed.. It increases test time much